framework/transport/ieee1905_transport/ieee1905_transport_local_bus.cpp

```diff
@@ -156,20 +159,36 @@ bool Ieee1905Transport::send_packet_to_local_bus(Packet &packet)
     return true;
 }
 
+static void copy_string(char *dst, const char *src, size_t dst_len)
+{
+    const char *src_end = std::find((char *)src, ((char *)src) + dst_len, '\0');
```

Comment:
Why are you casting away the constness of src? 
(I'm surprised cppcheck doesn't report it)

Reply:
I have no idea - I just copied this from BCL, this is how its implemented there. I will fix it in both places

Reply:
Done - wasn't so simple since I had to link BCL to mapf::common and update all callers to use the new API location, but its done now. We need to move everything that should be shared to mapf::common eventually - so that BCL will be merged into mapf::common.